neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/Neo4jProperties.java

```diff
@@ -54,6 +54,14 @@
 	 */
 	public static final String ESTABLISH_CONNECTION_TIMEOUT = "hibernate.ogm.neo4j.client.establish_connection_timeout";
 
+	/**
+	 * The size of the client connection pool used by the RestEasy HTTP client
+	 * <p>
+	 *     Default value is 10
+	 * </p>
+	 */
+	public static final String REST_CONNECTION_POOL_SIZE = "hibernate.ogm.neo4j.client.rest.connection_pool_size";
```

Comment:
I would prefer not to mention the term REST, let's just remove it. Same for all the other places

Reply:
or maybe just use http instead of rest